docs/developer/PROW_CI.md

```diff
@@ -84,56 +350,55 @@ To update the Go version used in CI jobs of a branch:
 To update an OCP version in a branch:
 - Rename the oldest OCP version config file to the new version and change the occurrences in the file, and update jobs file.
 - Update links in README file.
+- Update envtest Kubernetes version in Makefile.
 
-Example: update `4.11` to `4.14` in master branch.
+Example: update `4.13` to `4.16` in master branch.
 
-`openshift-oadp-operator-master__4.11.yaml` needs to be renamed `openshift-oadp-operator-master__4.14.yaml`, in `ci-operator/config/openshift/oadp-operator/` folder of https://github.com/openshift/release repo.
+`openshift-oadp-operator-master__4.13.yaml` needs to be renamed `openshift-oadp-operator-master__4.16.yaml`, in `ci-operator/config/openshift/oadp-operator/` folder of https://github.com/openshift/release repo.
 
-`4.11` occurrences in the file need to be updated to `4.14`.
+`4.13` occurrences in the file need to be updated to `4.16`.
 ```diff
 ...
  releases:
    latest:
      release:
        channel: fast
--      version: "4.11"
-+      version: "4.14"
+-      version: "4.13"
++      version: "4.16"
  resources:
    '*':
      limits:
 ...
    branch: master
    org: openshift
    repo: oadp-operator
--  variant: "4.11"
-+  variant: "4.14"
+-  variant: "4.13"
++  variant: "4.16"
```

Comment:
This won't always work, one needs to find out which release stream exists. E.g. this 4.16 was available only within:

```shell
releases:
  latest:
    candidate:
      product: ocp
      stream: nightly
      version: "4.16"
```

In the past I've been using the following part to find out if the correct release is within correct stream + grepping other projects and looking which stream does the ocp belongs to:

https://amd64.ocp.releases.ci.openshift.org/api/v1/releasestream/4.16.0-0.nightly/latest

https://amd64.ocp.releases.ci.openshift.org/

Reply:
added note, will be in next commit